scripts/security_data_copy.py

```diff
@@ -0,0 +1,17 @@
+from distutils.dir_util import copy_tree
+from argparse import ArgumentParser
+
+parser = ArgumentParser()
+parser.add_argument('--source_path', type=str)
+parser.add_argument('--security_destination', type=str)
+parser.add_argument('--experiment', type=str)
+
+destination_path_name = '{}/experiment_{}'
+
+args = parser.parse_args()
+source_path = args.source_path
+security_destination = args.security_destination
+experiment = args.experiment
+destination_path = destination_path_name.format(security_destination, experiment)
+
+copy_tree(source_path, destination_path)
```

Comment:
Please, could you add an empty line below here? [Link to further reading](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)

```suggestion
copy_tree(source_path, destination_path)

```